locales/en/manage-kafka-permissions.json

```diff
@@ -72,6 +76,8 @@
     "topic": "Topic",
     "transactional_id": "Transactional ID"
   },
+  "review_existing_title": "Review existing permissions",
+  "selected_account_help": "Review the list of existing permissions for the selected account. The list includes account-specific permissions and permissions applied to all accounts within this Kafka instance. Permissions labeled<1>All accounts</1> cannot be removed when an individual account ID is selected.",
```

Comment:
You're missing a space in front of "All accounts"

<img width="258" alt="image" src="https://user-images.githubusercontent.com/21063328/198585366-ecc709a1-5dc4-4ada-acc0-118de0f13bc1.png">